ygen/enumgen_test.go

```diff
@@ -36,7 +105,7 @@ func TestFindEnumSet(t *testing.T) {
 		wantCompressed          map[string]*yangEnum
 		wantUncompressed        map[string]*yangEnum
 		wantSame                bool // Whether to expect same compressed/uncompressed output
-		wantErr                 bool
+		wantErrSubstr           string
```

Comment:
(no action required) I found [this article](https://blog.golang.org/go1.13-errors) on new error approaches in Go 1.13 pretty interesting - we might want to think about whether we want to experiment with these differences going forward. I don't think we need to right now at all - but I figured it might also be of interest.

Reply:
Interesting, maybe `ytypes.Validate` and `ytypes.Unmarshal`'s errors should be typed to allow for wrapping.